corehq/ex-submodules/dimagi/utils/couch/migration_operations.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management import call_command
+from django.db.migrations import RunPython
+
+
+class ReindexCouchViews(RunPython):
```

Comment:
If this reindexes both Couch and ES, I think the name is a bit confusing, and could be misleading if someone does not realize they are also causing an ES reindex to occur.

Reply:
I named it this way and put it in this location because the Elasticsearch reindex logic is in the process of being moved (@joxl is planning to do that very soon). It didn't feel worth it to use a different name for such a short period of time, although I did note it in the docstring.

Reply:
Agree that the naming is a bit misleading.

I see we are also calling `ptop_es_manage`, should that be added as a part of the later PR? I feel it would be better to add that with Joel's PR then when that is added.

Oh this PR is now merged.